readme.md

```diff
@@ -20,7 +20,8 @@ Silo dashboards are available by default on `localhost:8080` unless configured o
 
 | solution | docker support | clustering | clustered silos | grains | clients |
 | --- | --- | --- | --- | --- | --- |
-| [1](1/readme.md) | silo | localhost | 1 | 0 | 0 |
-| [2](2/readme.md) | - | localhost | 1 | 1 C# grain | console |
-| [3](3/readme.md) | silo, client | localhost | 1 | 1 C# grain | console |
-| [5](5/readme.md) | silo, client | localhost | 1 | 1 C# grain interfacing F# library code | console |
\ No newline at end of file
+| [1](1/readme.md) | silo | localhost | - | 0 | 0 |
+| [2](2/readme.md) | - | localhost | - | 1 C# grain | console |
+| [3](3/readme.md) | silo, client | localhost | - | 1 C# grain | console |
+| [4](4/readme.md) | silo, client | localhost | n | 1 C# grain | (n) console |
+| [5](5/readme.md) | silo, client | localhost | - | 1 C# grain interfacing F# library code | console |
```

Comment:
Not sure what happened with my previous comment, but only in example 4 we actually use clustering (in-memory cluster membership). In examples 1, 2, 3, 5, we should ideally give `-` as the clustering mode.